spec/api/v2/screening_lists/consolidated_spec.rb

```diff
@@ -308,5 +308,33 @@
       end
     end
 
+    context 'when expiration_date is specified' do
+      subject { response }
+      let(:params) { { sources: 'SDN', expiration_date: '[* TO 2011-01-23]' } }
```

Comment:
We need to be careful about what we allow to pass through here. See the warning in the [wildcard section](http://www.elastic.co/guide/en/elasticsearch/reference/1.x/query-dsl-query-string-query.html#_wildcards) of ES's documentation. If users can enter wildcards and we pass that on directly to the DB, they may put unacceptable load on the DB. What about ES's equivalent of [SQL injection](http://en.wikipedia.org/wiki/SQL_injection)?